requirements-dev.txt

```diff
@@ -23,3 +26,5 @@ jieba==0.39
 # other
 google-cloud-storage==1.7.0
 azure-storage-blob==1.0.0
+coveralls==1.7.0
```

Comment:
What do you think of referencing `alt_requirements/requirements_full.txt` (these already include spacy, MITIE)?

Reply:
From what I can see, the `alt_requirements/requirements_pretrained_embeddings_spacy.txt` doesn't include the EN/DE models installation, so refering to `requirements_full.txt` won't help us.

Reply:
> doesn't include the EN/DE models installation

True, but it includes stuff like Mitie, spacy which are currently both in the dev requirements and in the full-requirements (the full requirements reference the other requirements in alt requirements)

Reply:
You're right, I could **remove** the following lines from `requirements-dev.txt`:
```
git+https://github.com/tmbo/MITIE.git
spacy==2.1.4
jieba==0.39
```

And add:
```
-r alt_requirements/requirements_full.txt
```

Which should result in the exact same stuff getting installed. This would also remove the duplication of requirements listed: `jieba` is mentioned in `requirements-dev` and `requirements_full`, while `spacy` is mentioned in `requirements-dev` and `requirements_pretrained_embeddings_spacy`.

The models would need to be installed via a separate call to `pip install`.